.buildkite/scripts/steps/functional/scalability_dataset_extraction.sh

```diff
@@ -13,7 +13,7 @@ GCS_BUCKET="gs://kibana-performance/scalability-tests"
 .buildkite/scripts/bootstrap.sh
 
 echo "--- Extract APM metrics"
-scalabilityJourneys=("login" "promotion_tracking_dashboard")
+journeys=("login" "ecommerce_dashboard" "flight_dashboard" "web_logs_dashboard" "promotion_tracking_dashboard" "many_fields_discover")
```

Comment:
This code is not iterating this array, it is iterating the now non-existent `scalabilityJourneys` array

Reply:
My bad :( fixed in e8a2162db95bcc0dbbf0ad0f606d8a29448d79aa